src/index.tsx

```diff
@@ -1,13 +1,33 @@
 import { createRoot } from 'react-dom/client';
-import { HashRouter as Router } from 'react-router-dom';
+import {
+  Navigate,
+  Route,
+  HashRouter as Router,
+  Routes,
+} from 'react-router-dom';
 
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { App } from './App';
+import { PeoplePage } from './components/PeoplePage';
+import { PeopleProvider } from './Context';
 
 createRoot(document.getElementById('root') as HTMLDivElement).render(
-  <Router>
-    <App />
-  </Router>,
+  <PeopleProvider>
+    <Router>
+      <Routes>
+        <Route path="/" element={<App />}>
+          <Route path="/home" element={<Navigate to="/" replace={true} />} />
+          <Route path="/people" element={<PeoplePage />}>
+            <Route path="/people/:slug" />
+          </Route>
```

Comment:
The nested route `<Route path='/people/:slug' />` is currently not rendering any component. If you intend to display a specific component when a person is selected, you should provide an `element` prop with the desired component. Otherwise, consider removing this route if it's not needed.